src/tests/asserts.go

```diff
@@ -21,3 +24,14 @@ func AssertEqualValues(t *testing.T, expected, actual interface{}) {
 			"actual  : %v", expected, actual)
 	}
 }
+
+func AssertEmpty(t *testing.T, actual interface{}) {
+	actualType := reflect.TypeOf(actual).Name()
```

Comment:
what do you think add t.Helper() on all assert functions to print stack trace correctly...? @esequielvirtuoso

Reply:
Done.